### PR TITLE
doc: Update search-over-new-field.md

### PR DESCRIPTION
**Pull request by igbopie**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: docs/how/search-over-new-field.md [hunk 1]
```diff
@@ -284,8 +298,43 @@ Response:
...
-      
+
     ]
   }
 }
 ```
+
+# Appendix: MidTier and UI changes
+
+## 1 Add fields to facets in MidTier if desired (optional)
```
Comment (jywadhwani) [inline docs/how/search-over-new-field.md:309]: let's put a `.` after each of the bullets i.e. `## 1.`?

File: docs/how/search-over-new-field.md [hunk 2]
```diff
@@ -284,8 +298,43 @@ Response:
...
     "total": 1,
     "links": [
-      
+
     ]
   }
 }
 ```
+
+# Appendix: MidTier and UI changes
```
Comment (jywadhwani) [inline docs/how/search-over-new-field.md:307]: One concern here is the context shift from the story shared in the backend section. We specifically took an example of adding a new search field `courses` in `CorpUserInfo` but then we are moving to dataset in the mid tier section. Do you think we can continue with the above example in this section?

**Review by jywadhwani — COMMENTED** [review]
Thanks for adding!

File: docs/how/search-over-new-field.md (hunk 2 above)
Comment (igbopie) [reply docs/how/search-over-new-field.md:307]: Sounds good

**Pull request closed without merging**